nimsuggest/tests/tinclude.nim

```diff
@@ -1,8 +1,10 @@
 discard """
-disabled:true
 $nimsuggest --tester compiler/nim.nim
->def compiler/semexprs.nim:25:50
-def;;skType;;ast.PSym;;PSym;;*ast.nim;;707;;2;;"";;100
->def compiler/semexprs.nim:25:50
-def;;skType;;ast.PSym;;PSym;;*ast.nim;;707;;2;;"";;100
+>def compiler/semexprs.nim:28:50
+def;;skType;;ast.PSym;;PSym;;*compiler/ast.nim;;727;;2;;"";;100
+>def compiler/semexprs.nim:28:50
+def;;skType;;ast.PSym;;PSym;;*compiler/ast.nim;;727;;2;;"";;100
 """
+
+# Notes:
+# This test is really slow, likely useful for profiling nimsuggest
```

Comment:
would be fixed via https://github.com/nim-lang/Nim/pull/16401/files#r546276926

Reply:
Fixed this and discovered some fun new stateful behaviour.

I didn't realize how far it would stretch, but I'm guessing the compile time evaulation leaves things in an awkward state at times. Basically, definition look breaks depending upon where you look at it from. 😅